spec/requests/api/v1/articles_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe Api::V1::ArticlesController, type: :request do
+  let(:headers) { { HTTP_ACCEPT: 'application/json' } }
+
+  describe 'GET /api/v1/articles/:id' do
+    let!(:existing_entries) do
+      create(:article,
+            id: 1,
+            title: 'Breaking News',
+            body: 'Some breaking content')
+    end
+
+    before do
+      get '/api/v1/articles/1', headers: headers
+    end
+
+    it 'returns a 200 response status' do
+      expect(response).to have_http_status 200
+    end
+
+    it 'returns an article' do
+      expect(response_json.count).to eq 1
+    end
+
+    it 'returns article title' do
+      expect(response_json["entries"]["title"]).to eq "Breaking News"
+    end
+
+  end
```

Comment:
what happens with a request when there is no article with the `id` being sent?

Reply:
sad path has been put in place